src/Goerli-DssSpellCollateral.sol

```diff
@@ -17,6 +17,9 @@
 pragma solidity 0.6.12;
 
 import "dss-exec-lib/DssExecLib.sol";
+import "dss-interfaces/dss/GemJoinAbstract.sol";
+import "dss-interfaces/dss/IlkRegistryAbstract.sol";
```

Comment:
Forgive me if I am ignorant here, but are IlkRegistryAbstract and GemAbstract used here? I can see GemJoinAbstract being used explicitly - is there a requirement for these two?